e2e/helpers/docker_compose.py

```diff
@@ -0,0 +1,62 @@
+import os
+import shlex
+import subprocess
+from time import sleep
+
+
+def stop():
+    subprocess.check_call("docker-compose stop -t 0".split(' '))
```

Comment:
unfortunately not. We can shorten it though, only using `.split()` - but that would be less explicit.